### PR TITLE
Add stacked bar chart

### PR DESCRIPTION
**Pull request by balajimanoharan**
Separate PR for stacked bar chart
Rebased on top of #15. Can be merged once we have the right metrics for the bar charts

Screen Shot 2020-05-05 at 2 51 38 PM

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/stacked-bar.js [hunk 1]
```diff
@@ -21,92 +21,92 @@ export default class StackedBarComponent extends Component {
       chart: {
         stacked: true,
```
Comment (balajimanoharan) [inline app/components/stacked-bar.js]:
```suggestion
      chart: {
        animations: {
          enabled: false
        },
        stacked: true,
```

File: app/components/stacked-bar.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+};
+
+export default class StackedBarComponent extends Component {
+  @service intl;
+
+  get chartOptions() {
+    const { intl } = this;
+    return {
+      chart: {
+        stacked: true,
```
Comment (balajimanoharan) [inline app/components/stacked-bar.js]:
```suggestion
      chart: {
        animations: {
          enabled: false
        },
        stacked: true,
```

**Review by balajimanoharan — COMMENTED** [review]
Should turn animations off in this as well

**Comment by kevinhinterlong** [comment]
Should make the colors consistent too

File: tests/integration/components/chart-container-test.js [hunk 3]
```diff
@@ -3,24 +3,36 @@ import { setupRenderingTest } from 'ember-qunit';
...
+    assert
+      .dom('.chart-container__timeseries .timeseries__title')
+      .hasText('Cases Over Time', 'the timeseries chart renders with a title');
+    assert
+      .dom('.chart-container .chart-container__timeseries .ember-apex-chart')
       .exists('A timeseries chart is also rendered in the chart container');
+
+    assert
+      .dom('.chart-container .chart-container__stacked-bar')
+      .exists('A timeseries is rendered in the chart container');
```
Comment (kevinhinterlong) [inline tests/integration/components/chart-container-test.js]:
```suggestion
      .exists('A stacked bar is rendered in the chart container');
```

File: app/components/chart-container.hbs [hunk 4]
```diff
@@ -31,7 +50,7 @@
           {{svg-jar "close" class="modal__close-icon m-r-3" width="20px" height="20px"}}
         </span>
         <div class="modal-header">
-          <h3>{{t (concat 'chartTitle.' this.modalChart)}}</h3>
+          <h3>{{@location.attributes.label}} {{t (concat 'chartTitle.' this.modalChart)}}</h3>
```
Comment (kevinhinterlong) [inline app/components/chart-container.hbs:53]: Should be `{{t (concat 'chartTitle.' this.modalChart) place=@location.attributes.label}}` for better intl support

**Review by kevinhinterlong — APPROVED** [review]

File: app/components/chart-container.hbs (hunk 4 above)
Comment (jkusa) [reply app/components/chart-container.hbs:53]: In non-modal form we use `{{t (concat 'chartTitle.' this.modalChart)}}` without the `place`

**Pull request merged**